lib/chef/resource/file.rb

```diff
@@ -52,78 +52,17 @@ class File < Chef::Resource
 
       def initialize(name, run_context=nil)
         super
-        @path = name
-        @backup = 5
-        @atomic_update = Chef::Config[:file_atomic_update]
-        @force_unlink = false
-        @manage_symlink_source = nil
-        @diff = nil
         @verifications = []
```

Comment:
Don't block this on my account, but it seems like `verifications` is the real property and `verify` is a setter (appender) method.